build.gradle

```diff
@@ -1,14 +1,16 @@
 
 
 buildscript {
-  repositories {
+    ext.kotlin_version = '1.3.30'
+    repositories {
```

Comment:
Use 2 space indents.

Reply:
I set my Groovy formatting options to be 2 spaces and the formatting of the entire file was changed. Is the 2 spaces style a post 2.4 convention?